www/i18n/en.json

```diff
@@ -166,6 +166,7 @@
         "replaces": "Replaces",
         "purpose": "Purpose",
         "survey": "Details",
+        "untracked-time-range": "Untracked: {{start}} - {{end}}",
```

Comment:
I didn't get to this today.

The "Timestamps are invalid" message should also be internationalized https://github.com/e-mission/e-mission-phone/pull/935#issuecomment-1489711407
The default Enketo error message ("Form contains errors. Please see fields marked in red.") does not appear to be internationalized either.

Reply:
I internationalized all 3 of these. 4be125a049ce44e8957e6a13268c6764acca4cc2  https://github.com/e-mission/e-mission-translate/pull/23

Because I was not sure, I targeted the PR at 'master'. Let me know if it needs to be targeted at 'master_for_platform' instead.

Reply:
I've rebased to `master_for_platform` and merged